tests/test_consumer/test_handlers/test_runs_handler_airflow.py

```diff
@@ -76,6 +77,10 @@ async def test_runs_handler_airflow(
     runs = run_scalars.all()
     assert len(runs) == 2
 
+    user_query = select(User).where(User.name == "myuser")
+    user_scalars = await async_session.scalars(user_query)
+    user = user_scalars.one_or_none()
```

Comment:
```suggestion
    user = user_scalars.one()
```

Reply:
It is one_or_none() for this case:
maybe our extracto is broken and it doesn't exctract user. So then user with name = 'myuser' will be None.
And in this case test will fail on this assertion `assert task_run.started_by_user_id is not None`

Reply:
Ok